########################################################################################################################
# Gait Parameters
########################################################################################################################

syropod:
  gait_parameters:

########################################################################################################################

    # Step order: AR,CL,BL,AL,CR,BR
    wave_gait:

      stance_phase: 10
      swing_phase: 2
      phase_offset: 2

      offset_multiplier: {AR: 2, BR: 3, CR: 4, CL: 1, BL: 0, AL: 5}

########################################################################################################################

    # Step order: AR/BL/CR,AL/BR/CL
    tripod_gait:

      stance_phase: 2
      swing_phase: 2
      phase_offset: 2

      offset_multiplier: {AR: 0, BR: 1, CR: 0, CL: 1, BL: 0, AL: 1}

########################################################################################################################

    # Step order: AR,CL,BR,AL,CR,BL (overlapping)
    ripple_gait:

      stance_phase: 4
      swing_phase: 2
      phase_offset: 1 

      offset_multiplier: {AR: 2, BR: 0, CR: 4, CL: 1, BL: 3, AL: 5}

########################################################################################################################

    # Step order: AR/CL,BR/BL,AL/CR
    amble_gait:

      stance_phase: 2
      swing_phase: 1
      phase_offset: 1

      offset_multiplier: {AR: 1, BR: 2, CR: 0, CL: 1, BL: 2, AL: 0}

########################################################################################################################
########################################################################################################################
